migrateTo_7_1_1/src/main/groovy/org/bonitasoft/migration/FillBeforeMigratingTo7_1_1.groovy

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (C) 2013 BonitaSoft S.A.
+ * BonitaSoft, 32 rue Gustave Eiffel - 38000 Grenoble
+ * This library is free software; you can redistribute it and/or modify it under the terms
+ * of the GNU Lesser General Public License as published by the Free Software Foundation
+ * version 2.1 of the License.
+ * This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth
+ * Floor, Boston, MA 02110-1301, USA.
+ **/
+package org.bonitasoft.migration
+import org.bonitasoft.engine.LocalServerTestsInitializer
+import org.bonitasoft.engine.api.PlatformAPIAccessor
+import org.bonitasoft.engine.api.TenantAPIAccessor
+import org.bonitasoft.engine.bpm.bar.BusinessArchiveBuilder
+import org.bonitasoft.engine.bpm.form.FormMappingModelBuilder
+import org.bonitasoft.engine.bpm.process.impl.ProcessDefinitionBuilder
+import org.bonitasoft.engine.exception.BonitaException
+import org.bonitasoft.engine.form.FormMappingSearchDescriptor
+import org.bonitasoft.engine.form.FormMappingTarget
+import org.bonitasoft.engine.form.FormMappingType
+import org.bonitasoft.engine.search.SearchOptionsBuilder
+import org.bonitasoft.engine.test.PlatformTestUtil
+import org.bonitasoft.migration.filler.FillAction
+import org.bonitasoft.migration.filler.FillerInitializer
+import org.bonitasoft.migration.filler.FillerShutdown
+import org.bonitasoft.migration.filler.FillerUtils
+
+import java.util.zip.ZipEntry
+import java.util.zip.ZipOutputStream
+/**
+ * @author Baptiste Mesta
+ */
+class FillBeforeMigratingTo7_1_1 {
+
+    @FillerInitializer
+    public void init() {
+        FillerUtils.initializeEngineSystemProperties()
+        LocalServerTestsInitializer.beforeAll();
+    }
+
+
+    @FillAction
+    public void fillSomething() {
+        def session = TenantAPIAccessor.getLoginAPI().login("install", "install");
+        def identityAPI = TenantAPIAccessor.getIdentityAPI(session)
+        identityAPI.createUser("john", "bpm")
+        TenantAPIAccessor.getLoginAPI().logout(session)
+    }
+
+
+
+
+    @FillAction
+    public void migrateFormMapping() {
```

Comment:
Is this step really necessary? As the migration does not impacts form mappings maybe we could deploy a very simple process.